.github/workflows/main.yml

```diff
@@ -0,0 +1,88 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches: [ test-ci ]
+  pull_request:
+    branches: [ test-ci ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    name: Build DCNM collection
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ansible: [2.9.12, 2.10.0]
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+    
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install ansible-base (v${{ matrix.ansible }})
+      run: pip install https://github.com/ansible/ansible/archive/v${{ matrix.ansible }}.tar.gz --disable-pip-version-check
```

Comment:
Sorry did not get this point. Why are we using pip install instead of git clone directly. Because this is how we do install on our local MAC. Is this because we want to get TGZ files for faster response

Reply:
Good question.  We could investigate using git clone in the future but for now I was following the same process from the ACI/MSO collection.  Not sure if this is faster but we just need a stable base ansible collection to be in place.